src/Translator.php

```diff
@@ -144,6 +144,18 @@ private function setModulePaths(): void
                     continue;
                 }
                 $branch = $this->exec('git rev-parse --abbrev-ref HEAD', $modulePath);
+                // the following happens when a tag is checked out, rather than a minor or next minor branch
+                if ($branch === 'HEAD') {
+                    $cmd = "git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/\\1/'";
```

Comment:
Please add a comment above this explaining what it's doing - I don't understand sed very well. There are two regex expressions in here and I'm confused.
A comment in the code will mean I won't be confused when I look at it in the future as well as during this review.